traversal/src/main/scala/overflowdb/traversal/Traversal.scala

```diff
@@ -202,9 +207,41 @@ class Traversal[A](elements: IterableOnce[A])
       }.getOrElse(Traversal.empty)
     }
 
+  /** aggregate all objects at this point into the given collection (side effect)
+   * e.g. `mutable.ArrayBuffer.empty[A]` */
```

Comment:
the example is slightly confusing. maybe you meant
```
val xs = mutable.ArrayBuffer.empty[A]
ys.aggregate(xs)
```
?

Reply:
true, updated